src/features/game/GameProvider.tsx

```diff
@@ -40,15 +48,30 @@ export const GameProvider: React.FC = ({ children }) => {
 
   // TODO - Typescript error
   const gameService = useInterpret(gameMachine) as MachineInterpreter;
-  const [shortcuts, setShortcuts] = useState<InventoryItemName[]>(
-    getShortcuts()
-  );
+  const inventory = useSelector(gameService, selectInventory);
+  const sessionId = useSelector(gameService, selectSessionId);
+  const [shortcuts, setShortcuts] = useState<InventoryItemName[]>([]);
 
   const shortcutItem = useCallback((item: InventoryItemName) => {
     const items = cacheShortcuts(item);
+
     setShortcuts(items);
   }, []);
 
+  useLayoutEffect(() => {
+    const savedShortcuts = getShortcuts();
+
+    if (sessionId !== INITIAL_SESSION && savedShortcuts.length === 0) {
```

Comment:
I don't think we can use `INITIAL_SESSION` anymore since each contract no longer uses `0x000...` as the initial session ID.

I imagine that `sessionId !== INITIAL_SESSION` will always be true. Consider omitting it.